controllers/grid/eventLog/EventLogGridRow.inc.php

```diff
@@ -25,11 +25,18 @@ class EventLogGridRow extends GridRow {
 	/** @var Submission **/
 	var $_submission;
 
+	/** @var boolean Is the current user assigned as an author to this submission */
+	var $_isCurrentUserAssignedAuthor;
+
 	/**
 	 * Constructor
+	 * @param Submission $submission
+	 * @param boolean $isCurrentUserAssignedAuthor Is the current user assigned
```

Comment:
very minor thing: first the variable then the type :-)

Reply:
Arg! I always do that!